src/core/lwip/tcp_in.c

```diff
@@ -176,6 +176,7 @@ void L3_level_tcp_input(struct pbuf *p, struct tcp_pcb *pcb)
             in_data.inseg.seqno = in_data.seqno;
             in_data.inseg.flags = 0;
             in_data.inseg.tcp_flags = in_data.flags;
+            in_data.inseg.bufs = 0;
```

Comment:
Do we set it to 0 because this is RX?

Reply:
Shouldn't it be 1 in such case?
does bufs say how much chained including the current one or excluding? Worth to add this info to the comment of this var.

Reply:
Will be implemented in a different PR.